company-meghanada.el

```diff
@@ -63,8 +63,8 @@
   (company-mode t)
   (set (make-local-variable 'company-backends) nil)
   (set (make-local-variable 'company-transformers) nil)
-  (set (make-local-variable 'company-idle-delay) 0)
-  (set (make-local-variable 'company-minimum-prefix-length) company-meghanada-prefix-length)
+  (when company-meghanada-prefix-length
+    (set (make-local-variable 'company-minimum-prefix-length) company-meghanada-prefix-length))
   (setq company-meghanada-trigger-regex (format company-meghanada--trigger company-meghanada-prefix-length company-meghanada-prefix-length))
```

Comment:
I see what you're saying, I pushed a change that sets `company-meghanada-prefix-length` to `company-minimum-prefix-length` if it is not set by the user.